string name
string subject
string object
float64 value  
bool is_symmetric
bool is_numeric #TODO: Need to have order information so we can do regular predicates too
